pt-BR/mail/chrome/messenger/safeMode.dtd

```diff
@@ -36,33 +36,32 @@
 #endif
    - ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY changeAndRestartButton2.label     "Save and Restart">
+<!ENTITY changeAndRestartButton2.label     "Salvar e reiniciar">
 <!ENTITY changeAndRestartButton2.accesskey "S">
 
 
 <!ENTITY safeModeOptionsWindow.width       "width: 49em;">
-<!ENTITY safeModeOptionsDialog.title       "&brandShortName; Trouble Shooting">
+<!ENTITY safeModeOptionsDialog.title       "Solução de Problemas do &brandShortName;">
 
-<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting 
-with one or more of these trouble shooting options selected:">
+<!ENTITY safeModeOptionsDescription2.label "Se você estiver com dificuldades no Postbox, tente reiniciar com uma dessas opções de solução de problemas selecionada:">
 
 
-<!ENTITY disableAddons.label              "Permamently Disable all Add-ons">
+<!ENTITY disableAddons.label              "Desabilitar permanentemente todos os Add-ons">
 <!ENTITY disableAddons.accesskey          "D">
 
-<!ENTITY resetToolbars.label              "Reset User Interface">
+<!ENTITY resetToolbars.label              "Redefinir a Interface de Usuário">
 <!ENTITY resetToolbars.accesskey          "R">
 
-<!ENTITY resetIndex.label                 "Reset Search Index">
+<!ENTITY resetIndex.label                 "Redefinir o índice de pesquisas">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY resetIndex.label                 "Redefinir o Índice de Pesquisas">